src/NUnitConsole/nunit3-console.tests/CommandLineTests.cs

```diff
@@ -43,6 +43,8 @@ public class CommandLineTests
         [TestCase("--arg1 @file1.txt --arg2 @file2.txt", "file1.txt:--fileArg1\n--fileArg2,file2.txt:--fileArg3", "--arg1 --fileArg1 --fileArg2 --arg2 --fileArg3", "")]
         [TestCase("--arg1 @file1.txt --arg2", "file1.txt:", "--arg1 --arg2", "")]
         [TestCase("--arg1 @file1.txt --arg2", "file1.txt:--fileArg1\n\n\n--fileArg2", "--arg1 --fileArg1 --fileArg2 --arg2", "")]
+        [TestCase("--arg1 @file1.txt --arg2", "file1.txt:--fileArg1\nThis is a COMMENT\n--fileArg2", "--arg1 --fileArg1 This is a COMMENT --fileArg2 --arg2", "")]
+        [TestCase("--arg1 @file1.txt --arg2", "file1.txt:--fileArg1\n#This is a COMMENT\n--fileArg2", "--arg1 --fileArg1 --fileArg2 --arg2", "")]
         public void ArgumentsFromFilesTests(string args, string files, string expectedExpandedArgs, string expectedErrorMessages)
```

Comment:
Should these use `Environment.NewLine` rather than just `\n`? (Not specific to this PR - I know!)